translation/it/README.md

```diff
@@ -0,0 +1,90 @@
+![AI Generativa per Principianti](../../img/1.png)
+
+### Un corso di 12 lezioni che ti insegnerà tutto quello che devi sapere per iniziare a costruire applicazioni con l'AI Generativa
```

Comment:
@godwin-17 Potresti sostituire 'AI' con 'IA' (Intelligenza artificiale) in tutto il documento?

Reply:
Suggerirei di sostituire 'costruire' con 'sviluppare/creare'